Added an introductory paragraph for new users (#536)
Macrometacorp/docs: docs/quickstart.md

2. **Faster Time-To-Market**: Macrometa is a converged technology Platform as a Service (PaaS) consisting of a multi-model database (K/V, document, search, graph), stream engine, complex event processing, and compute capabilities. Utilizing a single platform as opposed to stitching together a variety of services _a la carte_ in the public cloud allows you to build, iterate, and execute faster. Check out our [Edge Compute](https://www.macrometa.com/platform/edge-compute) to learn more.
3. **Lower Total Cost of Ownership (TCO)**: Building on the previous example, multiple cloud services are needed to build applications. When it's time to scale out to support a global audience, you'll end up utilizing several databases, maintaining copies of your data within multiple regions, and the costs will add up very quickly. Within the Macrometa GDN is a true [multi-model database](https://www.macrometa.com/topics/multi-model-database), allowing you to query one copy of data with extremely low latency anywhere in the world. Your development team will be free to spend more time on the business logic and less time focusing on the data layer and movement.

> Great, time to jump right in! Let's get started with Macrometa by creating a global address book. In this tutorial, we will be creating a document [collection](https://macrometa.com/docs/collections/), inserting and querying some user data, and finally creating a fully operational API via [Query Workers](https://macrometa.com/docs/queryworkers/). This tutorial is Macrometa's version of "Hello, world!", is simplistic by design, and demonstrates how you can quickly use our stateful serverless back-end to run a globally distributed database with local read-write latencies around 50ms. 

Reviewer: ```suggestion
Great, time to jump right in! Let's get started with Macrometa by creating a global address book. In this tutorial, you will be creating a document [collection](https://macrometa.com/docs/collections/), inserting and querying some user data, and finally creating a fully operational API via [Query Workers](https://macrometa.com/docs/queryworkers/). This tutorial is Macrometa's version of "Hello, world!", is simplistic by design, and demonstrates how you can quickly use our stateful serverless back-end to run a globally distributed database with local read-write latencies around 50ms. 
```

> If you've already completed this tutorial, you can skip ahead to [add search to your application](https://macrometa.com/docs/search/getting-started)

Reviewer: ```suggestion
If you've already completed this tutorial, then you can [add search to your application](https://macrometa.com/docs/search/getting-started).
```

> You have a full functional API for your application. [We made a front-end for you to take your new back-end for a spin](https://github.com/Macrometacorp/tutorial-addressbook-streams).

Reviewer: ```suggestion
You have a fully-functional API for your application. [We made a front-end for you to take your new back-end for a spin](https://github.com/Macrometacorp/tutorial-addressbook-streams).
```
